app/components/Table/index.js

```diff
@@ -20,6 +20,7 @@ type columnProps = {
   dataIndex: string,
   title: string,
   sorter?: (any, any) => number,
+  filter?: Array<{ label: string, value: any }>,
```

Comment:
and then `Array<FilterItem>`

Reply:
is `boolean` a good replacement for `any`?

Reply:
In your current case it might be, but if the goal is that you're supposed to be able to filter anything at a later point, then `any` might be the best.